dev/com.ibm.ws.kernel.boot.core/src/com/ibm/ws/kernel/provisioning/ServiceFingerprint.java

```diff
@@ -28,46 +29,49 @@
  */
 public class ServiceFingerprint {
     private static volatile File fingerPrintFile;
-    private static Properties props = new Properties();
+    private static Map<String, String> props = new HashMap<>();
```

Comment:
Changed this from a thread safe Properties object to a non-thread safe HashMap.  Need to add external synchronization or simply wrap in java.util.Collections.synchronizedMap(Map<K, V>).  Or we need to be convinced this class is only ever accessed from a single thread.

Reply:
Why is fingerPrintFile volatile but props is not?  Why can we not make props final instead?